.github/workflows/document_ruby_version.yml

```
@@ -0,0 +1,43 @@
+name: Add Ruby version to changelog && prepare release
+run-name: "Add ${{ inputs.is_jruby && 'J' || ''}}Ruby ${{ inputs.ruby_version }} to the CHANGELOG.md and prepare a release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      ruby_version:
+          description: "The Ruby version to announce"
+          type: string
+          required: true
+      is_jruby:
+          description: "JRuby version (as opposed to MRI)"
```

(The existing description here sounded like the field was for a version string rather than a bool)

```suggestion
          description: "JRuby release? (as opposed to MRI)"
```

---

Thanks! Reminded me of this scene https://www.youtube.com/watch?v=X3zfP14pLxc